Configuration/Eras/python/Era_Run3_UPC_2023_cff.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_UPC_cff import Run3_UPC
+from Configuration.Eras.Modifier_run3_egamma_2023_cff import run3_egamma_2023
+
+Run3_UPC_2023 = cms.ModifierChain(Run3_UPC, run3_egamma_2023)
```

Comment:
A large fraction of Eras are named `RunX_YYYY_mod` is it practical to create a bubble in the implementation of the pp era (here you re-implement the `Run3` -> `Run3_2023` detail).  ... why do we need `Run3_UPC`?

Reply:
Not needed at the moment, so I will remove Run3_UPC